app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :likes, dependent: :destroy
+  has_many :comments, dependent: :destroy
+
+  def most_recent_comments
+    comments.limit(5).order(created_at: :desc)
+  end
+
+  private
+
+  def update_post_counter
+    user.increment!(:post_counter)
+  end
```

Comment:
- [ ]  Let's use the `author` instead of the user 🤔 since you are using the author_id as the foreign key 👍🏻 

- [ ] Great job creating a method that updates the posts counter for a user. 👏🏻 👏🏻  However It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) -* `after_save` to call the `#update_post_counter` method every time a new post has been saved to database. You can make the `update_post_counter` method to be a `private` method 👍🏻 💯 
- 🤔  Your code should look like this 👇🏻 :


   ```ruby
     # app/models/post.rb
     after_save :update_post_counter
   
     private
   
     def update_post_counter
       author.increment!(:posts_counter)
     end
   ```

- [ ] Make sure you to use on line 13 `author.increment!` instead of `  user.increment!`